tests/features/page.feature

```diff
@@ -26,3 +26,16 @@ Feature: Page content creation
     And I should see "Teaser text"
     And I should see the link "financing"
     And I should see the link "European Patent Office"
+
+  @javascript
+  Scenario: Length limited fields are not allowing characters above the limit configured.
+    Given I am logged in as a user with the "create oe_page content, access content, edit own oe_page content, view published skos concept entities" permission
+    When I visit "the Page creation page"
+    Then I should see the text "Content limited to 150 characters, remaining: 150"
+    And I should see the text "Content limited to 250 characters, remaining: 250"
+    When I fill in "Title" with "My page"
+    And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
+    And I fill in "Teaser" with "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin eu hendrerit lacus, vitae bibendum odio. Fusce orci purus, hendrerit a magna at nullam. Text to remove"
+    And I fill in "Summary" with "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas felis leo, lobortis non eros in, consequat tempor est. Praesent sit amet sem eleifend, cursus arcu ac, eleifend nunc. Integer et orci sagittis, volutpat felis sit amet, tincidunt amet. Text to remove"
+    And I press "Save"
+    Then I should not see "The text to remove."
```

Comment:
Same as above.

Reply:
It will trim the extra characters. The only way to have more characters then the limit if we do copy paste. By typing you can't go over the limit, it will simply not type.